go/worker/storage/committee/node.go

```diff
@@ -200,6 +202,31 @@ func NewNode(
 
 	n.ctx, n.ctxCancel = context.WithCancel(context.Background())
 
+	go func() {
```

Comment:
Isn't doing this async dangerous? Where do we ensure that start is not called before this completes? Related goroutines should probably block on a channel that gets closed once this deferred init completes?